### DISCUSSION TITLE
Controlling bitrate over AirPlay

### DISCUSSION THREAD

**Discussion by danteodosiu** [philippe44/AirConnect#473] [General]
Hi Philippe,

First, I'd like to thank you for putting together AirConnect - it's the only way I found to stream Qobuz from my Mac to my (older) DLNA-enabled Cyrus Stream XP and it works great!!

I have a question about bitrate, hope you can help. The AirPlay device from AirConnect shows up on my Mac as having a 44100Hz format. I think the Hi-Res output from Qobuz (e.g. for 96000Hz music) gets downsampled to 44100Hz, meaning it's essentially streaming CD-quality to the Cyrus instead of the full Hi-Res.

Is there any way to control this to get the full Hi-Res rate? The rate=xxx parameter that can be used for PCM in the config file doesn't seem to be working for FLAC.

Thanks!

Dan

**Comment by philippe44** [comment]
Thanks Dan. AirPlay 1 as a standard is limited to 44.1kHz
